### PR TITLE
Add tests for transit_model_procmacro

### PR DESCRIPTION
**Pull request by woshilapin**

### CODE REVIEW

File: transit_model_procmacro/Cargo.toml [hunk 1]
```diff
@@ -5,10 +5,22 @@ authors = ["Guillaume Pinot <texitoi@texitoi.eu>"]
...
 [dependencies]
 syn = "0.11.11"
 quote = "0.3.15"
+
+[dev-dependencies]
+pretty_assertions = "0.6"
+trybuild = "1"
+transit_model = { path = "../" }
+transit_model_collection = { path = "../collection" }
+transit_model_relations = { path = "../relations" }
```
Comment (datanel) [inline transit_model_procmacro/Cargo.toml]: it's very painfull to release cyclic dependencies

Comment (woshilapin) [reply transit_model_procmacro/Cargo.toml]: Even for `dev-dependencies`?

Comment (datanel) [reply transit_model_procmacro/Cargo.toml]: yes

Comment (woshilapin) [reply transit_model_procmacro/Cargo.toml]: OK, so then, what do we do?

Comment (woshilapin) [reply transit_model_procmacro/Cargo.toml]: Will [this change in `cargo@1.40.0`](https://github.com/rust-lang/cargo/pull/7333) help with the problem?

Comment (datanel) [reply transit_model_procmacro/Cargo.toml]: according to the unit test yes

**Pull request merged**